### ISSUE TITLE
Wrong day name on Date Picker

### ISSUE THREAD

**Issue by KarlSayle** [buefy/buefy#2774]
### Description

The wrong day name ('M'onday, 'T'uesday, 'W'ednesday etc) is displayed above the date numbers. For example, today is Wednesday, 19/08/2020 but the date picker reports that it is Thursday (all day names appear to be shunted along by 1). I have been using the picker for a few months and have only spotted this today.

If it helps I am in UTC+1 (UK) timezone.

### Steps to reproduce

This is happening both on my app but also on the official documentation. See https://buefy.org/documentation/datepicker/

### Expected behavior

Today should show 'W' for Wednesday.

### Actual behavior

The date picker reports that today is Thursday, 'T'.

**Comment by KarlSayle** [comment]
After spotting @jtommy Twitter post regarding v0.9 and the breaking changes I checked which version we where using and it turns out we didn't have the version locked (naughty I know!). After locking the version to 0.8.20 the date picker started working as expected.

I'm sure this is already in hand but does the documentation need updating taking into account the new version too?

**Comment by Youdaman** [comment]
I've just come across this but can't find the tweet sorry.

I noticed that the day header shows "M T W T F S S" by default and also if I explicitly use `:first-day-of-week='0'`

If I try `:first-day-of-week='1'` then I get "W T F S S M T" which has shifted things by two instead of one.

I see that after I have reverted Buefy to v0.8.2 it correctly shows "Su M Tu W Th F S" by default.

I didn't see anything in the docs about this but might have missed it? That said the behaviour with 0.9.1 described above doesn't make sense to me with the offset being out by default.
